### PR TITLE
docs: Core Color class

### PR DESCRIPTION
**Pull request by Ombuweb**

### CODE REVIEW

File: content/guide/core/color.md [hunk 1]
```diff
@@ -0,0 +1,388 @@
+---
+title: Color
+description: Create a color object with any color representation and use it in your NativeScript apps.
+---
+
+This class allows you to create a color object with all color components(alpha, red, green, blue in the [0, 255] range) from different types of color representations such as, ARGB, color name, hex, etc. The properties `ios` and `android` of an instance of this class provide the color object for the respective native platform.
```
Comment (ashutosh887) [inline content/guide/core/color.md]: This class enables the creation of a color object with color components (alpha, red, green, blue) ranging from 0 to 255, using various color representations such as ARGB, color names, hex values, and more. The properties ios and android of an instance of this class provide the corresponding color object for the respective native platforms.

File: content/guide/core/color.md [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+
+---
+
+### android
+
+```ts
+androidColor = color.android
+```
+
+Gets the android-specific integer value representation. Same as the Argb one.
```
Comment (ashutosh887) [inline content/guide/core/color.md]: `ARGB` one

**Review by ashutosh887 — CHANGES_REQUESTED** [review]

File: content/guide/core/color.md (hunk 1 above)
Comment (NathanWalker) [inline content/guide/core/color.md]:
```suggestion
This `Color` class enables the creation of a color object with color components (alpha, red, green, blue) ranging from 0 to 255, using various color representations such as ARGB, color names, hex values, and more. The properties `ios` and `android` of an instance of this class provide the corresponding color object for the respective native platforms such as [UIColor](https://developer.apple.com/documentation/uikit/uicolor) on iOS and [Color ints](https://developer.android.com/reference/android/graphics/Color) on Android.
```

File: content/guide/core/color.md [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
```
Comment (NathanWalker) [inline content/guide/core/color.md]:
```suggestion
Gets the android-specific integer value representation. Same as the `ARGB` one.
```

File: content/guide/core/color.md [hunk 4]
```diff
@@ -0,0 +1,388 @@
+---
+title: Color
+description: Create a color object with any color representation and use it in your NativeScript apps.
+---
+
+This `Color` class enables the creation of a color object with color components (alpha, red, green, blue) ranging from 0 to 255, using various color representations such as ARGB, color names, hex values, and more. The properties `ios` and `android` of an instance of this class provide the corresponding color object for the respective native platforms such as [UIColor](https://developer.apple.com/documentation/uikit/uicolor) on iOS and [Color ints](https://developer.android.com/reference/android/graphics/Color) on Android.
```
Comment (rigor789) [inline content/guide/core/color.md]:
```suggestion
The `Color` class enables the creation of a color object:
 - using color components (alpha, red, green, blue) ranging from 0 to 255,
 - using various color representations like ARGB, color names, hex values, and more.
 
The `ios` and `android` properties of the `Color` class instance return the native [UIColor](https://developer.apple.com/documentation/uikit/uicolor) on iOS and [Color](https://developer.android.com/reference/android/graphics/Color) on Android.
```

Note: `using various color representations like ARGB, color names, hex values, and more.` we should define `and more` or remove it.

Suggestion:
 - split each option as a new line/list item and link them within the page. ie -> color names should link to a section about using color names etc.

**Pull request merged**